examples/ftd_api_scripts/timerange-access-rule-association.py

```diff
@@ -0,0 +1,163 @@
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "timeRangeObjects": [time_range],
+        "ruleAction": "TRUST",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    access_rule = post_access_rule(host, port, access_token, access_rule, parent_policy_id)
+    if not access_rule:
+        print('Unable to create access rule')
+        return False
+    access_rule_update = {
+        "version": access_rule['version'],
+        "name": "Test_Python_Request_For_Access_Rule__6.6.0_Updated_1",
+        "ruleId": access_rule['ruleId'],
+        "sourceZones": [],
+        "destinationZones": [],
+        "sourceNetworks": [],
+        "destinationNetworks": [],
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "timeRangeObjects": [time_range],
+        "ruleAction": "PERMIT",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    updated_access_rule = update_access_rule(host, port, access_token, access_rule['id'], access_rule_update,
+                                             parent_policy_id)
+    if not updated_access_rule:
+        print('Unable to update access rule')
+        return False
+    access_rule = get_access_rule(host, port, access_token, updated_access_rule['id'], parent_policy_id)
+    if not access_rule:
+        print('Unable to get access rule')
+        return False
+    print('Access rule name is {}'.format(access_rule['name']))
+    access_rule_update = {
+        "version": access_rule['version'],
+        "name": "Test_Python_Request_For_Access_Rule_Deleted_TRO_6.6.0_1",
+        "ruleId": access_rule['ruleId'],
+        "sourceZones": [],
+        "destinationZones": [],
+        "sourceNetworks": [],
+        "destinationNetworks": [],
+        "sourcePorts": [],
+        "destinationPorts": [],
+        "ruleAction": "PERMIT",
+        "eventLogAction": "LOG_BOTH",
+        "identitySources": [],
+        "users": [],
+        "logFiles": "false",
+        "destinationDynamicObjects": [],
+        "sourceDynamicObjects": [],
+        "type": "accessrule"
+    }
+    updated_access_rule = update_access_rule(host, port, access_token,
+                                             access_rule['id'], access_rule_update, parent_policy_id)
+    if not updated_access_rule:
+        print('Unable to update access rule')
+        return False
+    print('Access rule name is {}'.format(updated_access_rule['name']))
+    return True
+
+
+if __name__ == '__main__':
+
+    if len(sys.argv) != 6:
+        print("Usage: python cookbook_scripts/timerange-access-rule-association.py host port user passwd")
```

Comment:
parentId is a required param but is not shown in usage. Similar problem in timerange_access_rule_association_actions() where the parentid param is not included in docstring. But isn't the parentId for access policy hardcoded to 'default'? Why is the user required to pass it into main/timerange_access_rule_association_actions()? Should it just be hardcoded in timerange_access_rule_association_actions() ?

Reply:
Done